tests/python/unittest/test_auto_scheduler_evolutionary_search.py

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+""" Test evolutionary search. """
+
+import tvm
+from tvm import te, auto_scheduler
+
+from test_auto_scheduler_common import conv2d_nchw_bn_relu_auto_scheduler_test
+
+def test_evo_search():
```

Comment:
I suggest we add an extra test that uses SketchPolicy with XGBModel in `test_auto_scheduler_search_policy.py`, this should trigger the complete work flow including `GenerateSketches`, `SampleInitPopulation` and `EvolutionarySearch`.

Reply:
Hmm...it would become an end-to-end test instead to a unit test so I'm not sure if it's a good idea, especially we already have unit tests for those modules. However, I will try to add some more tests for each rule.